src/main/java/teammates/logic/api/EmailGenerator.java

```diff
@@ -685,4 +694,17 @@ private EmailWrapper getEmptyEmailAddressedToEmail(String recipient) {
         return email;
     }
 
+    private String getCoOwnersEmailsList(List<InstructorAttributes> instructors) {
```

Comment:
You may want to consider renaming the method as well, `getCoOwnersEmailsList` may seem to a reader that it returns a `List` instead of a `String`. Possibly `generateCoOwnersEmailsLine`.